diff -up kde-gtk-config-5.27.2/kconf_update/gtktheme.cpp.1~ kde-gtk-config-5.27.2/kconf_update/gtktheme.cpp
--- kde-gtk-config-5.27.2/kconf_update/gtktheme.cpp.1~	2023-02-28 13:17:06.000000000 +0100
+++ kde-gtk-config-5.27.2/kconf_update/gtktheme.cpp	2023-03-01 01:37:19.653143471 +0100
@@ -51,7 +51,7 @@ void upgradeGtk3Theme()
 
 QString gtk2Theme()
 {
-    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     QFile gtkrc(gtkrcPath);
     if (gtkrc.open(QIODevice::ReadWrite | QIODevice::Text)) {
         const QRegularExpression regExp(QStringLiteral("gtk-theme-name=[^\n]*($|\n)"));
diff -up kde-gtk-config-5.27.2/kded/config_editor/gtk2.cpp.1~ kde-gtk-config-5.27.2/kded/config_editor/gtk2.cpp
--- kde-gtk-config-5.27.2/kded/config_editor/gtk2.cpp.1~	2023-02-28 13:17:06.000000000 +0100
+++ kde-gtk-config-5.27.2/kded/config_editor/gtk2.cpp	2023-03-01 01:37:48.232320179 +0100
@@ -37,9 +37,9 @@ void replaceValueInGtkrcContents(QString
 
 void setValue(const QString &paramName, const QVariant &paramValue)
 {
-    QString gtkrcPath = qEnvironmentVariable("GTK2_RC_FILES", QDir::homePath() + QStringLiteral("/.gtkrc-2.0"));
+    QString gtkrcPath = qEnvironmentVariable("GTK2_RC_FILES", QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde"));
     if (gtkrcPath.contains(QStringLiteral(":/"))) { // I.e. env variable contains multiple paths
-        gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+        gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     }
     QFile gtkrc(gtkrcPath);
     QString gtkrcContents = Utils::readFileContents(gtkrc);
@@ -51,7 +51,7 @@ void setValue(const QString &paramName,
 
 void removeLegacyStrings()
 {
-    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     QFile gtkrc(gtkrcPath);
     QString gtkrcContents = Utils::readFileContents(gtkrc);
     if (gtkrcContents.isNull()) {
