diff -up kde-gtk-config-5.16.90/src/appearancegtk2.cpp.0000~ kde-gtk-config-5.16.90/src/appearancegtk2.cpp
--- kde-gtk-config-5.16.90/src/appearancegtk2.cpp.0000~	2019-09-19 12:40:43.000000000 +0200
+++ kde-gtk-config-5.16.90/src/appearancegtk2.cpp	2019-09-19 20:29:51.045935082 +0200
@@ -102,7 +102,7 @@ bool AppearanceGTK2::saveSettingsPrivate
 
         return true;
     } else {
-        qWarning() << "There was unable to write the .gtkrc-2.0 file";
+        qWarning() << "There was unable to write the .gtkrc-2.0-kde file";
         return false;
     }
 }
@@ -157,7 +157,7 @@ void AppearanceGTK2::reset()
 
 QString AppearanceGTK2::defaultConfigFile() const
 {
-    return QDir::homePath()+"/.gtkrc-2.0";
+    return QDir::homePath()+"/.gtkrc-2.0-kde";
 }
 
 QStringList AppearanceGTK2::installedThemes() const
diff -up kde-gtk-config-5.16.90/src/appearencegtk.h.0000~ kde-gtk-config-5.16.90/src/appearencegtk.h
--- kde-gtk-config-5.16.90/src/appearencegtk.h.0000~	2019-09-19 12:40:43.000000000 +0200
+++ kde-gtk-config-5.16.90/src/appearencegtk.h	2019-09-19 20:28:10.465161930 +0200
@@ -30,7 +30,7 @@
 class AbstractAppearance;
 /**
  * This class is responsible of administrating the GTK themes. It loads the 
- * configurations from the .gtkrc-2.0 file.
+ * configurations from the .gtkrc-2.0-kde file.
  */
 class AppearenceGTK
 {
