diff -up kde-gtk-config-5.16.4/CMakeLists.txt.omv~ kde-gtk-config-5.16.4/CMakeLists.txt
--- kde-gtk-config-5.16.4/CMakeLists.txt.omv~	2019-07-30 15:52:36.885111546 +0200
+++ kde-gtk-config-5.16.4/CMakeLists.txt	2019-07-30 15:52:43.860046925 +0200
@@ -13,6 +13,7 @@ include_directories(
     ${CMAKE_SOURCE_DIR}
     ${CMAKE_SOURCE_DIR}/src
     ${CMAKE_SOURCE_DIR}/src/ui
+    /usr/include/harfbuzz
 )
 
 include(ECMSetupVersion)
diff -up kde-gtk-config-5.16.4/gtk3proxies/CMakeLists.txt.omv~ kde-gtk-config-5.16.4/gtk3proxies/CMakeLists.txt
