diff -up kde-gtk-config-5.19.90/kded/configeditor.cpp.1~ kde-gtk-config-5.19.90/kded/configeditor.cpp
--- kde-gtk-config-5.19.90/kded/configeditor.cpp.1~	2020-09-17 12:50:36.000000000 +0200
+++ kde-gtk-config-5.19.90/kded/configeditor.cpp	2020-09-17 17:42:35.782509016 +0200
@@ -80,9 +80,9 @@ void ConfigEditor::setGtk3ConfigValueXSe
 
 void ConfigEditor::setGtk2ConfigValue(const QString &paramName, const QVariant &paramValue)
 {
-    QString gtkrcPath = qEnvironmentVariable("GTK2_RC_FILES", QDir::homePath() + QStringLiteral("/.gtkrc-2.0"));
+    QString gtkrcPath = qEnvironmentVariable("GTK2_RC_FILES", QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde"));
     if (gtkrcPath.contains(QStringLiteral(":/"))) { // I.e. env variable contains multiple paths
-        gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+        gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     }
     QFile gtkrc(gtkrcPath);
     QString gtkrcContents = readFileContents(gtkrc);
@@ -126,7 +126,7 @@ QString ConfigEditor::gtk3ConfigValueSet
 
 void ConfigEditor::removeLegacyGtk2Strings()
 {
-    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     QFile gtkrc(gtkrcPath);
     QString gtkrcContents = readFileContents(gtkrc);
 
diff -up kde-gtk-config-5.19.90/kded/themepreviewer.cpp.1~ kde-gtk-config-5.19.90/kded/themepreviewer.cpp
