diff -up kde-gtk-config-5.27.0/kconf_update/gtktheme.cpp.omv~ kde-gtk-config-5.27.0/kconf_update/gtktheme.cpp
--- kde-gtk-config-5.27.0/kconf_update/gtktheme.cpp.omv~	2023-02-14 15:47:07.158357146 +0100
+++ kde-gtk-config-5.27.0/kconf_update/gtktheme.cpp	2023-02-14 15:47:09.716401028 +0100
@@ -51,7 +51,7 @@ void upgradeGtk3Theme()
 
 QString gtk2Theme()
 {
-    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     QFile gtkrc(gtkrcPath);
     if (gtkrc.open(QIODevice::ReadWrite | QIODevice::Text)) {
         const QRegularExpression regExp(QStringLiteral("gtk-theme-name=[^\n]*($|\n)"));
diff -up kde-gtk-config-5.27.0/kded/config_editor/gtk2.cpp.omv~ kde-gtk-config-5.27.0/kded/config_editor/gtk2.cpp
--- kde-gtk-config-5.27.0/kded/config_editor/gtk2.cpp.omv~	2023-02-14 15:46:41.555811584 +0100
+++ kde-gtk-config-5.27.0/kded/config_editor/gtk2.cpp	2023-02-14 15:46:51.938058354 +0100
@@ -37,9 +37,9 @@ void replaceValueInGtkrcContents(QString
 
 void setValue(const QString &paramName, const QVariant &paramValue)
 {
-    QString gtkrcPath = qEnvironmentVariable("GTK2_RC_FILES", QDir::homePath() + QStringLiteral("/.gtkrc-2.0"));
+    QString gtkrcPath = qEnvironmentVariable("GTK2_RC_FILES", QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde"));
     if (gtkrcPath.contains(QStringLiteral(":/"))) { // I.e. env variable contains multiple paths
-        gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+        gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     }
     QFile gtkrc(gtkrcPath);
     QString gtkrcContents = Utils::readFileContents(gtkrc);
@@ -51,7 +51,7 @@ void setValue(const QString &paramName,
 
 void removeLegacyStrings()
 {
-    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     QFile gtkrc(gtkrcPath);
     QString gtkrcContents = Utils::readFileContents(gtkrc);
 
