diff -Naur kde-gtk-config-2.0.orig/src/appearancegtk2.cpp kde-gtk-config-2.0/src/appearancegtk2.cpp
--- kde-gtk-config-2.0.orig/src/appearancegtk2.cpp	2012-03-05 19:02:20.000000000 +0100
+++ kde-gtk-config-2.0/src/appearancegtk2.cpp	2012-04-11 01:59:25.000000000 +0200
@@ -82,7 +82,7 @@
     gtkrc.remove();
 
     if(!gtkrc.open(QIODevice::WriteOnly | QIODevice::Text)) {
-        kDebug() << "There was unable to write the file .gtkrc-2.0";
+        kDebug() << "There was unable to write the file .gtkrc-2.0-kde";
         return false;
     }
 
@@ -122,13 +122,14 @@
     gtkrc.close();
     
     //TODO: do we really need the linked file?
-    if(QFile::remove(gtkrcFile+"-kde4"))
+    if(QFile::remove(gtkrcFile+"4")){
         kDebug() << "ready to create the symbolic link";
-    
-    if( !QFile::link(gtkrcFile, gtkrcFile+"-kde4") )
-        kDebug() << "Couldn't create the symboling link to .gtkrc-2.0-kde4 :(";
-    else
-        kDebug() << "Symbolic link created for .gtkrc-2.0-kde4 :D";
+        if( !QFile::link(gtkrcFile, gtkrcFile+"4") )
+            kDebug() << "Couldn't create the symboling link to .gtkrc-2.0-kde4 :(";
+        else
+            kDebug() << "Symbolic link created for .gtkrc-2.0-kde4 :D";
+    }
+
     
     if(gtkrcFile==defaultConfigFile())
         KProcess::startDetached(KStandardDirs::findExe("reload_gtk_apps"));
@@ -138,7 +139,7 @@
 
 QString AppearanceGTK2::defaultConfigFile() const
 {
-    return QDir::homePath()+"/.gtkrc-2.0";
+    return QDir::homePath()+"/.gtkrc-2.0-kde";
 }
 
 QStringList AppearanceGTK2::installedThemes() const
diff -Naur kde-gtk-config-2.0.orig/src/appearencegtk.h kde-gtk-config-2.0/src/appearencegtk.h
--- kde-gtk-config-2.0.orig/src/appearencegtk.h	2012-03-05 19:02:20.000000000 +0100
+++ kde-gtk-config-2.0/src/appearencegtk.h	2012-04-11 01:40:22.000000000 +0200
@@ -30,7 +30,7 @@
 class AbstractAppearance;
 /**
  * This class is responsible of administrating the GTK themes. It loads the 
- * configurations from the .gtkrc-2.0 file.
+ * configurations from the .gtkrc-2.0-kde file.
  */
 class AppearenceGTK
 {
