diff -up kde-gtk-config-5.18.0/kded/configeditor.cpp.omv~ kde-gtk-config-5.18.0/kded/configeditor.cpp
--- kde-gtk-config-5.18.0/kded/configeditor.cpp.omv~	2020-02-08 23:58:27.318415353 +0100
+++ kde-gtk-config-5.18.0/kded/configeditor.cpp	2020-02-08 23:58:41.334415971 +0100
@@ -82,9 +82,9 @@ void ConfigEditor::setGtk3ConfigValueXSe
 
 void ConfigEditor::setGtk2ConfigValue(const QString &paramName, const QString &paramValue)
 {
-    QString gtkrcPath = qEnvironmentVariable("GTK2_RC_FILES", QDir::homePath() + QStringLiteral("/.gtkrc-2.0"));
+    QString gtkrcPath = qEnvironmentVariable("GTK2_RC_FILES", QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde"));
     if (gtkrcPath.contains(QStringLiteral(":/"))) { // I.e. env variable contains multiple paths
-        gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+        gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     }
     QFile gtkrc(gtkrcPath);
     QString gtkrcContents = readFileContents(gtkrc);
@@ -97,7 +97,7 @@ void ConfigEditor::setGtk2ConfigValue(co
 
 QString ConfigEditor::gtk2ConfigValue(const QString& paramName)
 {
-    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     QFile gtkrc(gtkrcPath);
     if (gtkrc.open(QIODevice::ReadWrite | QIODevice::Text)) {
         const QRegularExpression regExp(paramName + QStringLiteral("=[^\n]*($|\n)"));
@@ -126,7 +126,7 @@ QString ConfigEditor::gtk3ConfigValueSet
 
 void ConfigEditor::removeLegacyGtk2Strings()
 {
-    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+    QString gtkrcPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
     QFile gtkrc(gtkrcPath);
     QString gtkrcContents = readFileContents(gtkrc);
 
diff -up kde-gtk-config-5.18.0/kded/themepreviewer.cpp.omv~ kde-gtk-config-5.18.0/kded/themepreviewer.cpp
--- kde-gtk-config-5.18.0/kded/themepreviewer.cpp.omv~	2020-02-08 23:58:46.384416193 +0100
+++ kde-gtk-config-5.18.0/kded/themepreviewer.cpp	2020-02-08 23:58:54.167416536 +0100
@@ -36,8 +36,8 @@
 #include "configeditor.h"
 #include "config.h"
 
-const QString ThemePreviewer::previewGtk2ConfigPath = QStandardPaths::writableLocation(QStandardPaths::TempLocation) + QStringLiteral("/gtkrc-2.0");
-const QString ThemePreviewer::currentGtk2ConfigPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0");
+const QString ThemePreviewer::previewGtk2ConfigPath = QStandardPaths::writableLocation(QStandardPaths::TempLocation) + QStringLiteral("/gtkrc-2.0-kde");
+const QString ThemePreviewer::currentGtk2ConfigPath = QDir::homePath() + QStringLiteral("/.gtkrc-2.0-kde");
 const QString ThemePreviewer::gtk2PreviewerExecutablePath = QStandardPaths::findExecutable(QStringLiteral("gtk_preview"), {CMAKE_INSTALL_FULL_LIBEXECDIR});
 
 const QString ThemePreviewer::gtk3PreviewerExecutablePath = QStandardPaths::findExecutable(QStringLiteral("gtk3_preview"), {CMAKE_INSTALL_FULL_LIBEXECDIR});
