diff -Naur kde-gtk-config-5.5.3/src/appearancegtk2.cpp kde-gtk-config-5.5.3.tpg/src/appearancegtk2.cpp
--- kde-gtk-config-5.5.3/src/appearancegtk2.cpp	2016-01-06 12:00:16.000000000 +0000
+++ kde-gtk-config-5.5.3.tpg/src/appearancegtk2.cpp	2016-01-16 04:53:20.753785611 +0000
@@ -92,7 +92,7 @@
     gtkrc.remove();
 
     if(!gtkrc.open(QIODevice::WriteOnly | QIODevice::Text)) {
-        qWarning() << "There was unable to write the file .gtkrc-2.0";
+        qWarning() << "There was unable to write the file .gtkrc-2.0-kde";
         return false;
     }
 
@@ -132,10 +132,10 @@
     gtkrc.close();
     
     //TODO: do we really need the linked file?
-    if(QFile::remove(gtkrcFile+"-kde4"))
+    if(QFile::remove(gtkrcFile+"4"))
         qDebug() << "ready to create the symbolic link";
     
-    if( !QFile::link(gtkrcFile, gtkrcFile+"-kde4") )
+    if( !QFile::link(gtkrcFile, gtkrcFile+"4") )
         qWarning() << "Couldn't create the symboling link to .gtkrc-2.0-kde4 :(";
 //     else
 //         qDebug() << "Symbolic link created for .gtkrc-2.0-kde4 :D";
@@ -148,7 +148,7 @@
 
 QString AppearanceGTK2::defaultConfigFile() const
 {
-    return QDir::homePath()+"/.gtkrc-2.0";
+    return QDir::homePath()+"/.gtkrc-2.0-kde";
 }
 
 QStringList AppearanceGTK2::installedThemes() const
diff -Naur kde-gtk-config-5.5.3/src/appearencegtk.h kde-gtk-config-5.5.3.tpg/src/appearencegtk.h
--- kde-gtk-config-5.5.3/src/appearencegtk.h	2016-01-06 12:00:16.000000000 +0000
+++ kde-gtk-config-5.5.3.tpg/src/appearencegtk.h	2016-01-16 04:49:12.840232040 +0000
@@ -30,7 +30,7 @@
 class AbstractAppearance;
 /**
  * This class is responsible of administrating the GTK themes. It loads the 
- * configurations from the .gtkrc-2.0 file.
+ * configurations from the .gtkrc-2.0-kde file.
  */
 class AppearenceGTK
 {
